web/src/edit/NeighbourhoodMode.svelte

```diff
@@ -374,213 +401,238 @@
     </nav>
   </div>
   <div slot="sidebar">
-    <h2>Editing tools</h2>
     <div
-      class="tool-palette"
-      style="display: flex; justify-content: space-between; flex-wrap: wrap; gap: 6px;"
+      style="display: flex; justify-content: space-between; align-items: center"
     >
+      <h2>Editing tools</h2>
+      <label>
```

Comment:
Happy for other ideas about how to style / place the toggle and to show the editing controls are disabled
![Screenshot from 2025-06-13 08-44-04](https://github.com/user-attachments/assets/02dc3e09-4862-49d3-867b-90d60dd6c264)